ansible/roles/software/ckan/inventory/molecule/default/molecule.yml

```diff
@@ -14,14 +14,6 @@ platforms:
     image: ubuntu:trusty
     groups:
       - trusty
-  - name: ckan-inventory-app-bionic
-    image: ubuntu:bionic
```

Comment:
🤔   I wouldn't have removed this but it's correct that this configuration will never be run on bionic. I'd prefer to test all roles on both trusty and bionic where possible. That way we don't have to think about is this role supported on trusty vs bionic (less thinking or knowledge about the code is better).

Are you seeing that this scenario no longer works on bionic?

Reply:
I think I was trying to be too efficient on tests, and have different scenarios for classic and next. Instead, I'm going to slightly refactor to pull back in bionic and only have next specific tests on the inventory-next scenario...